lib/adapters/file/read.js

```diff
@@ -9,7 +9,7 @@ var Read = Juttle.proc.source.extend({
     procName: 'read-file',
 
     initialize: function(options, params, location, program) {
-        var allowed_options = ['from', 'to', 'file', 'timeField', 'format'];
+        var allowed_options = ['from', 'to', 'file', 'timeField', 'format', 'formatOptions'];
```

Comment:
Agreed as that would make things look simpler after all but its just odd that `-pattern` then applies to the format code and not the `read xyz` code.